src/core/server/saved_objects/import/create_saved_objects.ts

```diff
@@ -103,14 +105,15 @@ export const createSavedObjects = async <T>({
     const bulkCreateResponse = await savedObjectsClient.bulkCreate(objectsToCreate, {
       namespace,
       overwrite,
+      workspaces,
     });
     expectedResults = bulkCreateResponse.saved_objects;
   }
 
   // remap results to reflect the object IDs that were submitted for import
   // this ensures that consumers understand the results
   const remappedResults = expectedResults.map<CreatedObject<T>>((result) => {
-    const { id } = objectIdMap.get(`${result.type}:${result.id}`)!;
+    const { id } = objectIdMap.get(`${result.type}:${result.id}`) || ({} as SavedObject<T>);
```

Comment:
so there will be case that get return null? It seems ! is to assure there won't be such a case and will throw exception if get return null, do we want to change this?

Reply:
I remember that there is an error while doing unit test, will revert this line to see if unit test can pass as before.